app/services/work_version_event_description_builder.rb

```diff
@@ -1,24 +1,32 @@
 # frozen_string_literal: true
 
-# This creates an appropriate description for an update event on a WorkVersion
+# This creates an appropriate description for a create/update event on a WorkVersion
 class WorkVersionEventDescriptionBuilder
```

Comment:
this class will change to handle the new differing logic for new vs updated works